.github/workflows/ci.yml

```diff
@@ -92,11 +92,8 @@ jobs:
 
       - uses: dtolnay/rust-toolchain@master
         with:
-          # MSRV: Rust 1.47 and later have bugs in rustc that prevent some
-          # projects from upgrading past 1.46 yet. So, maintain compatibility
-          # for 1.46 until those bugs are fixed.
           # Make sure to update rust-version in Cargo.toml when updating MSRV.
-          toolchain: 1.46.0
+          toolchain: "1.60"
```

Comment:
In the commit message, you mention "specifically the edition in its `Cargo.toml`" but that would put the MSRV at 1.56. So presumably this is just the once_cell MSRV?

Reply:
Rewording the commit message. We had to go from 1.46 -> 1.56 to understand edition = "2021", and then one of the wasm dependencies was taking once_cell 1.x which got us 1.18 with a MSRV of 1.60